ui/ducks/bridge/actions.ts

```diff
@@ -11,26 +11,26 @@ import {
 import { forceUpdateMetamaskState } from '../../store/actions';
 import { submitRequestToBackground } from '../../store/background-connection';
 import { MetaMaskReduxDispatch } from '../../store/store';
+import { QuoteRequest } from '../../pages/bridge/types';
 import { bridgeSlice } from './bridge';
 
 const {
-  setToChainId: setToChainId_,
+  setToChainId,
   setFromToken,
   setToToken,
   setFromTokenInputValue,
   resetInputFields,
-  switchToAndFromTokens,
 } = bridgeSlice.actions;
 
 export {
-  setFromToken,
+  setToChainId,
+  resetInputFields,
   setToToken,
+  setFromToken,
   setFromTokenInputValue,
-  switchToAndFromTokens,
-  resetInputFields,
 };
 
-const callBridgeControllerMethod = <T>(
+const callBridgeControllerMethod = <T extends string | Partial<QuoteRequest>>(
```

Comment:
Can we just have it as `T`? Also 

`args?: T`. It would help make this fn more generic

Reply:
done! looks cleaner as well